inventory_management_system_api/routers/v1/item.py

```diff
@@ -51,6 +51,32 @@ def create_item(item: ItemPostRequestSchema, item_service: ItemService = Depends
         raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR, detail=message) from exc
 
 
+@router.get(path="/", summary="Get items", response_description="List of items")
+def get_items(
+    system_id: Annotated[Optional[str], Query(description="Filter items by system ID")] = None,
+    catalogue_item_id: Annotated[Optional[str], Query(description="Filter items by catalogue item ID")] = None,
+    item_service: Annotated[ItemService, None] = Depends(),
```

Comment:
![image](https://github.com/ral-facilities/inventory-management-system-api/assets/144466888/b9ec428f-0f5b-4ed1-abf8-2923571915d8)
I changed it to what you suggested, but it doesn't seem to like it

Reply:
Yeah that just means it has to be the first argument as the others have default parameters so it would need to be
item_service, system_id and catalogue_item_id.